### ISSUE TITLE
Add a mechanism for automatically loading settings

### ISSUE THREAD

**Issue by eth0up** [elespike/burp-cph#25]
The ability to automatically load settings would help when running Burp in headless mode, for automated deployments and the like. See the following post for a potential use case: https://support.portswigger.net/customer/portal/questions/17315873-headless-scan-in-burp-with-bearer-token

**Comment by elespike** [comment]
I'd also like to look into saving the global settings found in the Options tab, which control tool scope and verbosity.

**Comment by klaas-pggm** [comment]
Yes, please. The CPH extension is very usefull to me but I cannot use it in automation scenario's.
I would like to load a settings file from the command prompt in headless mode. Also I would like to have the scan option "Scanner" on the options tab set by default or loaded by configuration as well.
With those additions, this extension would be perfect for me.
Thanks a lot!

**Comment by elespike** [comment]
Decoupling CPH's logic from its UI will be much trickier than anticipated, unfortunately, to the point that it's probably easier to write a whole new module to handle headless operation.

I'll tinker with the idea, though it'll take some time.
